test/e2e_node/critical_pod_test.go

```diff
@@ -59,8 +58,14 @@ var _ = framework.KubeDescribe("CriticalPod [Serial] [Disruptive] [NodeFeature:C
 				framework.Skipf("unable to run test without dynamic kubelet config enabled.")
 			}
 			// because adminssion Priority enable, If the priority class is not found, the Pod is rejected.
-			systemCriticalPriority := &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: systemCriticalPriorityName}, Value: scheduling.SystemCriticalPriority}
-
+			// SystemPriorityClasses define system priority classes that are auto-created at cluster bootstrapping.
+			// SystemNodeCritical used for system critical pods that must not be moved from their current node.
+			// Because e2e_node.test does not start the scheduler, SystemNodeCritical needed to be created manually.
```

Comment:
these are created by the API server, not by the scheduler... are you sure we need to manually create these?

Reply:
Looks like you are correct @liggitt. Tests pass when this section (creating the PriorityClasses) is omitted, so I guess it should just be removed entirely.